### PR TITLE
Jessica Feng's CodingBat Part I

### PR DESCRIPTION
**Pull request by jskaa**
This is my codingbat assignment. 
It can be found at http://jskaa.github.io/spring2014/2014/02/08/jessicascodingbat1.html 

Cheers!

### CODE REVIEW

**Comment by eah13** [comment]
:boom:  nice work.

File: _posts/2014-02-8-jessicascodingbat1.md [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+```
+
+Don't speed, even if it is your birthday!
+
+###Logic I: cigar_party
+
+Truthfully, I just enjoyed this one because the thought of squirrels smoking cigars at a party was so silly it made me laugh. Writing the python for it was pretty simple, as it only required a simple if/else statement (There is very likely a more concise way, but I'm not sure). 
+
+My code was:
+```
```
Comment (eah13) [inline _posts/2014-02-8-jessicascodingbat1.md:75]: You need a blank line above this for the code block to render correctly

File: _posts/2014-02-8-jessicascodingbat1.md [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+    if is_weekend:
+        return cigars>=40
+    else:
+        return (cigars>=40) and (cigars<=60) 
+```
+
+Tadah!
+Anyways, I kept thinking of this:
+
+![Classy Squirrel](http://i.imgur.com/JdrPBrY.png "Dat Class")
```
Comment (eah13) [inline _posts/2014-02-8-jessicascodingbat1.md:86]: haha nice.  This is going into the Google community

**Comment by eah13** [comment]
Great discussion- fix that code block and you're ready to go

**Comment by jskaa** [comment]
@eah13 Fixed! (I'm pretty sure!)

**Comment by eah13** [comment]
Got it.

**Pull request merged**